base_layer/wallet/src/config.rs

```diff
@@ -125,7 +130,9 @@ pub struct WalletConfig {
     /// How many days do we need to start scanning before our actual birthday
     pub birthday_offset: u16,
     /// The URL of the HTTP client to use for base node requests
-    pub http_client_url: String,
+    pub http_server_url: String,
+    /// The fallback url address to use if the base node at http_server_url does not respond
+    pub fallback_http_server_url: String,
```

Comment:
I think we should rather add a `ConfigList<String>` here with multiple server entries